cookbook/configuration/configuration_organization.rst

```diff
@@ -34,7 +34,7 @@ default Symfony Standard Edition follow this structure:
 
 .. code-block:: text
 
-    <your-project>/
+    your-project/
```

Comment:
I read through it: it's focusing on *just* the stuff in the `app/` directory. I think really we never needed `src`, `vendor` or `web` here.